uh_internal/apps/dailyduties/ajax.py

```diff
@@ -28,33 +28,27 @@
 from ..core.templatetags.srs_urls import srs_edit_url
 from .models import DailyDuties, EmailViewingRecord
 from .utils import GetDutyData
-from .authhelper import get_token, get_admin_consent
 
 logger = logging.getLogger(__name__)
 
 
 @api_view(['GET'])
 def refresh_duties(request):
     duty_data = cache.get('duty_data')
-    admin_consent = get_admin_consent()
-    # token = get_token()
-    token = 1
```

Comment:
replace this with `service = setup()` from the utils.py file. then pass the service in below like token was being passed.